longclaw/shipping/api.py

```diff
@@ -20,25 +68,25 @@ def shipping_cost(request):
     fallback to the default shipping cost if it has been enabled in the app
     settings
     """
+    kwargs = get_shipping_cost_kwargs_or_response(request)
+    if isinstance(kwargs, Response):
```

Comment:
instead of checking the type and having `get_shipping_cost_kwargs_or_response` have ambiguous return types, we can just throw exceptions from get_shipping_cost_kwargs and catch them here. Something like
```
try:
  get_shipping_cost_kwargs(request)

except SomeExceptionTypeWithADictMessage as error:
   return Response(data=error.data, status=status.HTTP_400_BAD_REQUEST)
```

Reply:
That is definitely much cleaner